packages/plugin-analytic-fb-pixel/src/main.js

```diff
@@ -0,0 +1,41 @@
+import { ScriptLoaderPlugin } from 'ima-plugin-script-loader';
+import FacebookPixelAnalytic from './FacebookPixelAnalytic';
+
+let defaultDependencies = [
```

Comment:
The property $dependencies on class is better solution.

Reply:
Ok. Will change it in google-analytic also as it's copy-pasted from it.

Reply:
- Added `$dependencies` to AbstractAnalytic, GoogleAnalytic and FacebookPixelAnalytic.
- removed `initBind` method from script and README. Class is registered when user requests it via `oc.get()` in his app.